apps/web/src/modules/job/detail/pages/index.test.tsx

```diff
@@ -8,6 +8,14 @@ describe('<JobDetailPage />', () => {
     useRouter: jest.fn(),
   }))
   const useRouter = jest.spyOn(require('next/router'), 'useRouter')
+  jest.doMock('common/context/UserContext', () => ({
+    useUser: () => ({
+      user: {
+        isVerified: true,
+        type: UserType.ACTOR,
+      },
+    }),
+  }))
```

Comment:
Is `mockUser()` enough 
ref: https://pkhing.notion.site/Front-Mock-useUser-bde9aa10d8eb4747b242ce4b2009bafe

Reply:
Well yes, I have already forgotten this utility function 😂